docs/outlook/item-multi-select.md

```diff
@@ -21,8 +21,76 @@ Complete the [Outlook quick start](../quickstarts/outlook-quickstart.md?tabs=yeo
 
 ## Configure the manifest
 
+# [Unified manifest for Microsoft 365](#tab/jsonmanifest)
+
+1. In your preferred code editor, open the Outlook quick start project you created.
+
+1. Open the **manifest.json** file located at the root of the project.
+
+1. In the "authorization.permissions.resourceSpecific" array, change the value of the "name" property to "Mailbox.ReadWrite.User". It should look like the following when you're done.
+
+    ```json
+    "authorization": {
+        "permissions": {
+            "resourceSpecific": [
+                {
+                    "name": "Mailbox.ReadWrite.User",
+                    "type": "Delegated"
+                }
+            ]
+        }
+    },
+    ```
+
+1. In first object of the "extensions.runtimes" array, make the following changes.
+
+    1. Change the "requirements.capabilities.minVersion" property to "1.13".
+    1. Set the "actions.pinnable" property to `true`.
+    1. In the same "actions" object, add the "multiselect" property and set it to `true`.
+
+    Your code should look like the following after you've made the changes.
+
+    ```json
+    "runtimes": [
+        {
+            "requirements": {
+                "capabilities": [
+                    {
+                        "name": "Mailbox",
+                        "minVersion": "1.13"
+                    }
+                ]
+            },
+            "id": "TaskPaneRuntime",
+            "type": "general",
+            "code": {
+                "page": "https://localhost:3000/taskpane.html"
+            },
+            "lifetime": "short",
+            "actions": [
+                {
+                    "id": "TaskPaneRuntimeShow",
+                    "type": "openPage",
+                    "pinnable": true,
+                    "view": "dashboard",
+                    "multiselect": true
+                }
+            ]
+        },
+        ...
+    ]
+    ```
+
+1. Delete the second object of the "extensions.runtimes" array, whose "id" is "CommandsRuntime".
+
+1. In the "extensions.ribbons.tabs.controls" array, delete the second object, whose "id" is "ActionButton".
+
+1. Save your changes.
+
 > [!NOTE]
-> The item multi-select feature isn't currently supported in the [Unified manifest for Microsoft 365 (preview)](../develop/unified-manifest-overview.md), but the team is working on making this available.
+> Item multi-select can also be enabled without setting the "runtimes.actions.multiselect" property to `true` if the "runtimes.actions.supportsNoItemContext" property is set to `true` in the manifest. To learn more, see [Activate your Outlook add-in without the Reading Pane enabled or a message selected](contextless.md).
```

Comment:
This note is correct, but I wonder if doing this would be a good practice. Since the default for "multiselect" is false, normally the absence of it would imply that it is false. But if users follow the suggestion in this note, they'll have a manifest where "multiselect" is not present but is true. I think this would be misleading and potentially confusing to a developer who is looking at the manifest to troubleshoot some problem. If I'd known what the product team was planning with this property, I'd have argued against this design. But I guess that ship has sailed, so I'd recommend that we just not tell people about this **implicit** setting of "multiselect" to true. If they always explicitly include it and set it to true when it should be, then their manifest is self-documenting.

Reply:
That's a great point. Thanks for bringing it up. I removed this note.